sycl/test-e2e/Graph/Explicit/host_task_last.cpp

```diff
@@ -7,6 +7,9 @@
 
 // REQUIRES: aspect-usm_shared_allocations
 
+// https://github.com/intel/llvm/issues/14350
+// UNSUPPORTED: linux && gpu-intel-gen12
```

Comment:
Since this issue looks limited to the `SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1` run, i'd prefer to skip that specific run invocation rather than not running the test.

Reply:
sure, this should be addressed in latest commit, thx